### DISCUSSION TITLE
How to make the web ui work? asking for user and password...

### DISCUSSION THREAD

**Discussion by ddeslandes1** [euzu/tuliprox#55] [Q&A] [answered]
I am struggling to use the web ui, I have m3ufilter running in server mode on my pc, it works connecting to an IPTV source and filtering the content, but I want to log in the web ui and I don't know what should be the username and password for that..??

I kept the default "xt/xt.secret" from the use case example...

help!

**Comment by euzu** [comment, marked as answer]
@ddeslandes1  the webui has optional  credential support. You  can disable it.
The WebUi user are defined in a file user.txt (you can change the name)

Here is the wiki page

[web auth](https://github.com/euzu/m3u-filter/wiki/config_web_auth)

**Reply by ddeslandes1** [reply]
thanks will look it up

**Reply by jamesarbrown** [reply]
On my docker container I also found impatience causes it Login menu appear, even when not setup. I waited another few minutes and hit the url again and went straight in.

**Reply by euzu** [reply]
@jamesbrown it shouldn't take minutes,  when the backend is up and running the endpoint is available, if the endpoint is not available the ui cant interact with the backend,   i can write on my todo list  to show a screen which says backend not available try again later.
